### PR TITLE
fix: fix the jina versioning

### PR DESCRIPTION
**Pull request by nan-wang**
The `jina_version` is retrieved from the installed Jina when running hub-builder. Therefore, using `devel` image will lead to the version is always ahead of the released version.

Reference:
- In [action-hub-updater](https://github.com/jina-ai/action-hub-updater/blob/master/hub_updater.py#L353), the jina versioning is retrieved from [api.jina.ai/latest](). 
- Ihttps://github.com/jina-ai/jina/blob/master/jina/docker/hubio.py#L16

### CODE REVIEW

**Comment by nan-wang** [comment]
The test failed.

https://github.com/jina-ai/action-hub-builder/pull/14/checks?check_run_id=1634063676#step:4:1

```bash
  rc=0
  [[ "" == ".github/workflows/tests/ImageReader/" ]] || rc=1
  [[ "" == ".github/workflows/tests/EmptyExecutor/" ]] || rc=1
  exit $rc
  shell: /bin/bash -e {0}
Error: Process completed with exit code 1.
```

**Comment by JoanFM** [comment]
> The test failed.
> 
> https://github.com/jina-ai/action-hub-builder/pull/14/checks?check_run_id=1634063676#step:4:1
> 
> ```shell
>   rc=0
>   [[ "" == ".github/workflows/tests/ImageReader/" ]] || rc=1
>   [[ "" == ".github/workflows/tests/EmptyExecutor/" ]] || rc=1
>   exit $rc
>   shell: /bin/bash -e {0}
> Error: Process completed with exit code 1.
> ```

u must bump the version of the hub executors in the test. it is a badly designed test though.

**Comment by JoanFM** [comment]
> > The test failed.
> > https://github.com/jina-ai/action-hub-builder/pull/14/checks?check_run_id=1634063676#step:4:1
> > ```shell
> >   rc=0
> >   [[ "" == ".github/workflows/tests/ImageReader/" ]] || rc=1
> >   [[ "" == ".github/workflows/tests/EmptyExecutor/" ]] || rc=1
> >   exit $rc
> >   shell: /bin/bash -e {0}
> > Error: Process completed with exit code 1.
> > ```
> 
> u must bump the version of the hub executors in the test. it is a badly designed test though.

a change was made to make sure the manifest is bumped to avoid merging PR in the hub that do not bump manifest version that would missmatch docker image with github code

**Comment by nan-wang** [comment]
GA does NOT support for setting `--build-arg` when building docker images and therefore `jina_verison` can not be set properly.

Reference:
https://github.community/t/feature-request-build-args-support-in-docker-container-actions/16846/6

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,9 @@
-FROM jinaai/jina:devel
+FROM jinaai/jina:latest
 
 RUN apt-get update && apt-get install -y jq curl
 
+RUN pip install docker
```
Comment (deepankarm) [inline Dockerfile:5]: Is this required because image `jinaai/jina:latest` doesn't ship with `jina[docker]`?

**Review by deepankarm — APPROVED** [review]

**Comment by JoanFM** [comment]
should this be fixed inside hubio of core? or only handled in this actions?

**Comment by deepankarm** [comment]
> should this be fixed inside hubio of core? or only handled in this actions?

When we pull from `jinaai/jina:latest`, version would be pulled automatically inside hubio.

**Comment by JoanFM** [comment]
> > should this be fixed inside hubio of core? or only handled in this actions?
> 
> When we pull from `jinaai/jina:latest`, version would be pulled automatically inside hubio.

I mean to set it into the jina requirements.txt, I opened a PR in core for that

**Comment by JoanFM** [comment]
but not sure is the proper place to handle

**Comment by nan-wang** [comment]
> I mean to set it into the jina requirements.txt, I opened a PR in core for that

This is actually a good idea. It will solve the dilemma that we can't set arbitrary versioning in this GA due to the usage of jina base image. When building the pod image, the user might not want to use the `latest` version.

File: Dockerfile (hunk 1 above)
Comment (JoanFM) [reply Dockerfile:5]: yes it is

**Review by JoanFM — APPROVED** [review]

**Review by JoanFM — CHANGES_REQUESTED** [review]
I think this is missleading because the jina version will be based on the base image

**Review by JoanFM — APPROVED** [review]
ah no I see now the point

**Comment by nan-wang** [comment]
Indeed, this is a bit confusing. We actually have two Jina environments, which can be different. One is set in this GA for running `jina hub build`, the other is set in the `requirements.txt` for running the docker images in the container.

**Pull request merged**